modules/end-user-guide/pages/creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -1,5 +1,5 @@
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project"]
-:navtitle: Creating a workspace by importing the source code of a project
+:navtitle: Creating a workspace with your project's source code
```

Comment:
Keep the original. The language reviews don't include heading changes (if they are grammatically ok) .)